### ISSUE TITLE
TOC - GitHub theme - 1st level headline CSS, Marked compiler

### ISSUE THREAD

**Issue by almereyda** [simov/markdown-viewer#28]
The Marked compiler renders first level headlines (lines starting with a single `#`) as raw (blue) HTML links into the Table of Contents outline. This may be due to #29. Remark doesn't show this behaviour, either.

Only the GitHub and GitHub dark themes are affected.

**Comment by simov** [comment]
Hi @almereyda,

Can you post a short markdown snippet for each one of your issues, that way it will be much more easier for me to reproduce it. You can use the following syntax here:

<pre>
```markdown
# some markdown
*here*
```
</pre>

**Comment by almereyda** [comment]
A Markdown example for both issues could look like this:

```markdown
---
title: Headline level one
tags: [ "tag1", "tag2", "tag3" ]
description: 
unicode: —
---

> This is a stub.

# Headline level one

Paragraph, like lorem ipsum or whatever.

## Headline level two
```

**Comment by simov** [comment]
The problem stems from the fact that [Marked](https://github.com/chjj/marked) doesn't support [Markdown Front Matter](https://jekyllrb.com/docs/frontmatter/).

Does Remark work for you?

**Comment by simov** [comment]
The fix is published in version 3.2 - Markdown Front Matter is supported with the **Remark** compiler.
